tests/test_data.py

```diff
@@ -6,6 +6,22 @@
 from pgactivity.data import Data
 
 
+def wait_for_data(
+    fct, timeout: int = 2, desc_recursionError: str = "timeout in wait_for_data"
```

Comment:
I'd suggest `msg` instead of `desc_recursionError`; and drop the default value since all callers pass a value apparently.

Reply:
Fixed in 6e080de820bfff697476e45d0bcb69898cc967a6